crawlers/mooncrawl/mooncrawl/stats_worker/dashboard.py

```diff
@@ -411,6 +411,60 @@ def get_unique_address(
     )
 
 
+def get_blocks_state(
+    db_session: Session, blockchain_type: AvailableBlockchainType, address: str
+):
+
+    """
+    Generate meta information about
+    """
+
+    blocks_state = {
+        "blocks_state": {
+            "latest_block": {"address": 0, "labels": 0, "transactions": 0},
```

Comment:
This feels like too much nesting. Do we really need a `"blocks_state"` key? It is only needed inside the stats object.

Also, I would expect to just see something like this:
```
{
    "latest_available_block": 22119823,
    "latest_crawled_block": 22119756,
    "earliest_crawled_block": 11029941
}
```

Not sure what is value of `"labels"` and `"transactions"` and in general the schema you have seems too complicated.